organization.go

```diff
@@ -242,3 +261,50 @@ func (s *organizations) Delete(ctx context.Context, organization string) error {
 
 	return s.client.do(ctx, req, nil)
 }
+
+// Capacity shows the currently used capacity of an organization.
+func (s *organizations) Capacity(ctx context.Context, organization string) (*Capacity, error) {
+	if !validStringID(&organization) {
+		return nil, errors.New("Invalid value for organization")
+	}
+
+	u := fmt.Sprintf("organizations/%s/capacity", url.QueryEscape(organization))
+	req, err := s.client.newRequest("GET", u, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	c := &Capacity{}
+	err = s.client.do(ctx, req, c)
+	if err != nil {
+		return nil, err
+	}
+
+	return c, nil
+}
+
+// QueueOptions represents the options for showing the queue.
+type QueueOptions struct {
+	ListOptions
+}
+
+// Queue shows the current run queue of an organization.
+func (s *organizations) Queue(ctx context.Context, organization string, options QueueOptions) (*Queue, error) {
```

Comment:
I think we might want to call this `RunQueue` to be more clear. I could also see other types of queues being represented in the future (like an ingress queue, or a sentinel queue, etc.)

Reply:
👍